docker/test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+### Test script pulling the docker image and use it against showcase proto.
+
+# Docker image tag: gapic-generator-typescript:latest.
+# Create new directory showcase-typescript. 
+mkdir showcase-typescript
+# Pull the docker image
+docker pull gcr.io/gapic-images/gapic-generator-typescript:latest
```

Comment:
Here it makes sense to test local `gapic-generator-typescript` image before pushing to `gcr.io`, and not after. So don't pull here, just do `docker run ........ gapic-generator-typescript`. If it works, we do `docker push`.

Reply:
Sure! makes sense to me!